File: Teori Daspro/tugas6/soal1/C++/soal1.cpp
#include <iostream>
using namespace std;

int main() {
    int jml = 0 , i, n, angka;
    for (angka = 1; angka <= 300; angka++){
        n = 0;
        for (i = 2; i <= angka / 2; i++) {
            if (angka % i == 0) {
                n++;
                break;
            }
        }
        if (n == 0 && angka != 1) {
            cout << angka << ", ";
            jml += angka;
        }
    }
    cout << "Jumlah total = " << jml;
}

File: Teori Daspro/tugas6/soal1/Python/soal1.py
jml = 0
for angka in range(1, 300):
    n = 0
    for i in range(2, angka):
        if int(angka % i) == 0:
            n += 1
            break

    if n == 0 and angka != 1:
        print(angka, end=", ")
        jml += angka

print("Jumlah total = ", jml)
